### PR TITLE
Retention at rank 4

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

**Comment by kianenigma** [comment]
Will open the proposal once I get at least one approval from @bkchr, or other members.

File: evidence/kianenigma/0002-2024-12-retention-rank-4.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    * [Polkadot-SDK Features Overview in 2024](https://blog.kianenigma.com/talks/frame-2024/)
+* Exploring new FRAME ideas such as [FLITE](https://github.com/kianenigma/flite), which have been a [source of inspiration](https://github.com/OpenZeppelin/polkadot-runtime-templates/issues/257#issuecomment-2278749395) for [OpenZeppelin templates](https://github.com/OpenZeppelin/polkadot-runtime-templates/pull/300). 
+  
+
+## Voting record
+*Provide your voting record in relation to required thresholds for your rank.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|IV|48%   |100%  | 51 votes out of 107, leading to a 48% rate, starting from ref 148 onwards, which was my retention in June  | |
```
Comment (anaelleltd) [inline evidence/kianenigma/0002-2024-12-retention-rank-4.md:45]: |IV| 60% | 90% | 51 votes out of 107, leading to a 48% rate, starting from ref 148 onwards, which was my retention in June |  |

Comment (anaelleltd) [reply evidence/kianenigma/0002-2024-12-retention-rank-4.md:45]: Please do not edit the thresholds. Only fill in the 'Member's voting activities" and "Comment" columns with your data.

File: evidence/kianenigma/0002-2024-12-retention-rank-4.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+## Misc
+
+- Comment(s): 
+Being in a position of leadership within both Parity and the ecosystem, I want to acknowledge that some of the work that I named above is achieved with the help of other individuals, and I played the role of a mentor/coordinator. I am grateful to work with all of the with other Parity employees and Fellowship members, for example @ggwpez to name one.
+- I will strive to improve my voting rate, and apply for a promotion to Rank 5 in June 2025. I will still retain my part time salary only. My main intention to promote myself would be to eligible to vote on more proposals, and remove the burdeon on the current rank 5+ members for promotions to rank 3+, and so on.
```
Comment (joepetrowski) [inline evidence/kianenigma/0002-2024-12-retention-rank-4.md:52]:
```suggestion
- I will strive to improve my voting rate, and apply for a promotion to Rank 5 in June 2025. I will still retain my part time salary only. My main intention to promote myself would be to eligible to vote on more proposals, and remove the burdon on the current rank 5+ members for promotions to rank 3+, and so on. 
```

**Review by anaelleltd — APPROVED** [review]

**Review by ljuanto — APPROVED** [review]

File: evidence/kianenigma/0002-2024-12-retention-rank-4.md [hunk 3]
```diff
@@ -42,7 +42,7 @@
 
 |  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
 |---|---|---|---|---|
-|IV|48%   |100%  | 51 votes out of 107, leading to a 48% rate, starting from ref 148 onwards, which was my retention in June  | |
```
Comment (ljuanto) [inline evidence/kianenigma/0002-2024-12-retention-rank-4.md]: 60% | 90%

**Review by ljuanto — APPROVED** [review]

**Pull request merged**